fixedwidth/src/main/java/org/apache/metamodel/fixedwidth/FixedWidthConfiguration.java

```diff
@@ -33,66 +33,74 @@
 /**
  * Configuration of metadata about a fixed width values datacontext.
  */
-public final class FixedWidthConfiguration extends BaseObject implements
-		Serializable {
+public final class FixedWidthConfiguration extends BaseObject implements Serializable {
 
-	private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 1L;
 
-	public static final int NO_COLUMN_NAME_LINE = 0;
-	public static final int DEFAULT_COLUMN_NAME_LINE = 1;
+    public static final int NO_COLUMN_NAME_LINE = 0;
+    public static final int DEFAULT_COLUMN_NAME_LINE = 1;
 
-	private final String encoding;
-	private final int fixedValueWidth;
-	private final int[] valueWidths;
-	private final int columnNameLineNumber;
-	private final boolean failOnInconsistentLineWidth;
-	private final ColumnNamingStrategy columnNamingStrategy;
+    private final String encoding;
+    private final int fixedValueWidth;
+    private final int[] valueWidths;
+    private final int columnNameLineNumber;
+    private final boolean failOnInconsistentLineWidth;
+    private final boolean headerPresent;

```

Comment:
Hmm I'm really at a loss because deep inside I feel that this is a very particular thing to EBCDIC and therefore I cannot come up with a fitting generic name. And so maybe we should just give it a name that reveals that it is a particular thing. Such as `skipEbcdicHeader` or whatever. Or maybe it is less particular than that, but then I am still not sure what it is! :)

I would almost consider subtyping the whole configuration class to have a EbcdicConfiguration class instead of FixedWidthConfiguration.